tss-esapi/tests/integration_tests/error_tests/return_code_tests/tpm_tests/tpm_format_zero_warning_tests.rs

```diff
@@ -74,7 +74,7 @@ macro_rules! test_display_trait_impl {
                 TpmFormatZeroWarningResponseCode::from(TpmFormatZeroWarning::$zero_warning)
             ),
             $expected_error_message,
-            "BaseReturnCode with {} did not produce the expected error message",
+            "TpmFormatZeroWarningResponseCode with {} did not produce the expected error message",
```

Comment:
Just [recently learned](https://github.com/wiktor-k/testresult/pull/3/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R18) that [`std::any::type_name::<T>()`](https://doc.rust-lang.org/std/any/fn.type_name.html) can be used to convert type name to string at compile-time.

(Not necessarily suggesting this change here, just noting that this would be compile-time checked and would survive type rename).

Reply:
That's pretty cool, thanks for sharing!

Reply:
Nice I made use of it. Because it also gives the whole module path not just the type name.